### DISCUSSION TITLE
Any chance to get Motoman NX100 HP6 working with ROS2 ?

### DISCUSSION THREAD

**Discussion by navdotnetreqs** [Yaskawa-Global/motoros2#313] [Q&A] [answered]
Hi, would anyone be able to answer whether there'd be any chance to get a Motoman NX100 HP6 (2006) working with ROS 2?

**Comment by gavanderhoorn** [comment, marked as answer]
Neither MotoROS1 nor MotoROS2 support that controller.

So it won't work with anything we support here, nor over at `ros-industrial/motoman`.

It could be there are other ways to interface with that controller and/or someone else has made something for the NX100 controller, but I wouldn't know.

**Comment by navdotnetreqs** [comment]
Ok, thank you!
